src/plugins/console/public/services/settings.ts

```diff
@@ -42,8 +42,8 @@ enum SettingKeys {
   AUTOCOMPLETE_SETTINGS = 'autocomplete_settings',
   CONSOLE_POLLING = 'console_polling',
   POLL_INTERVAL = 'poll_interval',
-  IS_HISTORY_DISABLED = 'is_history_disabled',
-  IS_KEYBOARD_SHORTCUTS_DISABLED = 'is_keyboard_shortcuts_disabled',
+  IS_HISTORY_ENABLED = 'is_history_enabled',
```

Comment:
@sabarasaba  This PR looks good, just a one question. Should we somehow to create a migration mechanizm to correctly handle cases where the user has already set some value before?

Reply:
This is a very good point!. Im not sure theres a considerable amount of users using this so that it would become too disruptive if we just change it.

How do you feel about perhaps write it as `IS_HISTORY_ENABLED = 'is_history_disabled',` so that the code will remain more semantic and we can add a comment explaining that this was a previous setting and in order to not generate migration issues for our users we are going to use the old key value.

Reply:
Addressed with https://github.com/elastic/kibana/pull/140902/commits/f642d3992695468efd263c7966bf55e4c043626f. Created a method that runs on the constructor to migrate the old settings to the new ones, this will ensure that the old setting keys will be removed from the storage and the new ones will be added. WDYT? @sabarasaba @alexwizp